integration-helpers/src/mpc_network/mocks.rs

```diff
@@ -88,6 +88,14 @@ impl MpcNetwork for MockMpcNet {
         0
     }
 
+    async fn send_bytes(&mut self, _bytes: &[u8]) -> Result<(), MpcNetworkError> {
+        Ok(())
+    }
+
+    async fn receive_bytes(&mut self, _num_expected: usize) -> Result<Vec<u8>, MpcNetworkError> {
+        Err(MpcNetworkError::RecvError)
+    }
+
```

Comment:
Do we use these anywhere?

Reply:
No we don't use these methods of the mock object anywhere. I updated the `MpcNetwork` trait in the `mpc-ristretto` crate separately [here](https://github.com/renegade-fi/mpc-ristretto/commit/521d50c9d339df527597ca302768d78b30a69a07) and we just need to implement dummy implementations in the mock for interface parity. Should've made this more clear